CRM/Contribute/Selector/Search.php

```diff
@@ -402,6 +402,13 @@ public function &getRows($action, $offset, $rowCount, $sort, $output = NULL) {
         }
       }
 
+      $row['balance_due'] = CRM_Core_BAO_FinancialTrxn::getPartialPaymentWithType(
+        $result->contribution_id,
```

Comment:
So @eileenmcnaughton do you have site to test this on, or are you proposing that JMA create a test site and populate it. Personally I don't think this is an issue. Let's see what the total number of queries per search results page is for 25 results with and without the change, and the differential page load time on a server with reasonable resources, and then discuss with @colemanw ? Recall as well that we are on a deadline on this and we had agreement to move ahead.